src/key.js

```diff
@@ -133,7 +133,7 @@ module.exports = (common) => {
       it('does not contain the removed name', (done) => {
         ipfs.key.list((err, res) => {
           expect(err).to.not.exist()
-          const found = res.Keys.filter(k => k.Name === key.name)
+          const found = res.Keys.filter(k => k.Name === key.Name)
```

Comment:
I don't disagree with the lowercase .name. You can convert that in js-ipfs-api. It is friendlier to js devs.

Reply:
@diasdavid go-ipfs returns Capitalised names.  We must support it as is.

Reply:
Fine then